src/index.js

```diff
@@ -164,12 +164,22 @@ window.TurbolinksAnimate = window.TurbolinksAnimate || new function() {
         return;
       }
 
-      newElement.style[cssPropertyToCamelCase(property)] = getComputedStyle(element).getPropertyValue(property);
+      let properties = property.split(',');
```

Comment:
lI also think that we should change the name of the variable `property` to something more appropriate. Maybe we should call `split` directly in line 155, and assign the result to `properties` there. What do you think?